.azure-pipelines/integration-tests.yml

```diff
@@ -6,6 +6,7 @@ trigger:
     exclude:
       - docs/*
       - .github/*
+      - README.md
```

Comment:
This doesn't seem to work the way I wanted :( ...

Reply:
> Also, the path exclude parameter only works for file path instead of file

:( source: https://developercommunity.visualstudio.com/t/azure-pipelines-triggers-pathsexclude-in-pr-is-not/1046653#T-N1047494